<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="moveToRep">
        <property name="text">
         <string>Move to the folder with repository</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pullChanges">
        <property name="text">
         <string>Pull changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addNewChangesToIndex">
        <property name="text">
         <string>Add new changes/files to index</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PushChanges">
        <property name="text">
         <string>Push changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showCommitedChanges">
        <property name="text">
         <string>Show committed changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commitedChangesLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="pathToCloneLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="cloneNewRep">
        <property name="text">
         <string>Clone new repository</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commitNameLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="commitChanges">
        <property name="text">
         <string>Commit changes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="changeSearchLine"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchForChanges">
        <property name="text">
         <string>Search for changes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="showBranchList">
        <property name="text">
         <string>Show branch list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkoutToBranch">
        <property name="text">
         <string>Checkout to branch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ShowCommitList">
        <property name="text">
         <string>Show commit list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkoutToCommit">
        <property name="text">
         <string>Checkout to commit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>546</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
